app/jobs/consume_assessments_job.rb

```diff
@@ -216,7 +226,7 @@ def handle_opt_in_opt_out(message)
     BlockedNumber.create(phone_number: monitoree_number) if message['response_status'] == 'opt_out'
     BlockedNumber.where(phone_number: monitoree_number).destroy_all if message['response_status'] == 'opt_in'
     patients = Patient.responder_for_number(monitoree_number)
-    patients.uniq.each do |patient|
+    patients.each do |patient|
```

Comment:
This might be out of scope, but it's kinda weird that we do an each loop here at all. `Patient.responder_for_number(monitoree_number)` is only supposed to return 1 monitoree and there's no checks for that. Thoughts?

Reply:
This is an interesting question and I remember this being a point of contention when this was originally written.  `Patient.responder_for_number(monitoree_number)` should ideally return one number but in reality there might be instances of user error such as a monitoree was double added or a dependent was accidentally not put into a household but they have the same phone number as their responder -- Without context we need to make sure that assessments that we know came from a given number are consumed for any monitorees which they realistically could have been filled by.

Reply:
That's fair - we can just leave for now then. Thanks!